docs/src/creating-packages.md

```diff
@@ -0,0 +1,189 @@
+# **5.** Creating Packages
+
+A package is a project with a `name`, `uuid` and `version` entry in the `Project.toml` file `src/PackageName.jl` file that defines the module `PackageName`.
+This file is executed when the package is loaded.
+
+### Generating files for a package
+
+To generate files for a new package, use `pkg> generate`.
```

Comment:
May want to advertise PkgDev in this section, since the "packages" generated this way are kind of limited.